containers/ldap.docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM alpine
+
+MAINTAINER Max Neunhoeffer <max@arangodb.com>
+
+RUN apk add --update openldap openldap-back-mdb && \
+    mkdir -p /run/openldap /var/lib/openldap/openldap-data && \
+    rm -rf /var/cache/apk/* && mkdir /arangodbldif /arangodbcerts
+
+ENV ORGANISATION_NAME "ArangoDB GmbH"
+ENV SUFFIX "dc=arangodb,dc=com"
+ENV ROOT_USER "admin"
+ENV ROOT_PW "password"
+ENV USER_UID "neunhoef"
+ENV USER_GIVEN_NAME "Max"
+ENV USER_SURNAME "Neunhoeffer"
+ENV USER_EMAIL "max@arangodb.com"
```

Comment:
Should we use @neunhoef here? Maybe something neutral?

Reply:
done